apps/rule-manager/app/service/RuleManager.scala

```diff
@@ -12,11 +12,14 @@ import com.gu.typerighter.model.{
   RegexRule,
   TextSuggestion
 }
+import com.gu.typerighter.rules.BucketRuleResource
 import db.{DbRuleDraft, DbRuleLive}
 import db.DbRuleDraft.autoSession
 import scalikejdbc.DBSession
 
-object DbRuleManager extends Loggable {
+import scala.util.Try
+
+object RuleManager extends Loggable {
```

Comment:
Renamed to reflect the fact that this manages rules in general, not just DbRules, including handling publishing duties. I moved publishing logic in here to get it out of the controller and into somewhere more testable.